spark-common/src/main/js/gulpfile.js

```diff
@@ -71,39 +32,21 @@ gulp.task('tdd', ['compile-soy'], function(done) {
 
 });
 
-gulp.task('build', ['test', 'compile-css-to-js'], function() {
-
-    return gulp.src([
-        'node_modules/iframe-resizer/js/iframeResizer.js',
-        'src/spark-noconflict-header.js', 'target/build/*.js', 'src/spark-bootstrap.js'])
-        .pipe(replace('{{spark_gulp_build_version}}', version))
-        .pipe(concat('spark-dist.js'))
-        .pipe(gulp.dest('target/gen'));
+gulp.task('build', function(callback) {
```

Comment:
The frontend unit tests should also be run as a part of the build process
Though it might be make sense to remove at least some of them...